sentences/nb/light_HassTurnOn.yaml

```diff
@@ -1,8 +1,16 @@
-language: 'nb'
+language: "nb"
 intents:
   HassTurnOn:
     data:
-      - sentences: []
+      - sentences:
+          - "<skru_på> [<alle>] <lys> [<i_på>] <område>"
+          - "<skru_på> [<alle>] <område> <lys>"
+          - "<skru_på>  <område> [<alle>] <lys>"
+          - "<skru_på> [<alle>] <område>[s]<lys>"
```

Comment:
I did not intend additional spacing, as this can be said in one word. English "bed room lights" can be stated as "soveromslys" in Norwegian. The added (s) applies for soveromslys, but not kjøkkenlys.